dataflow/src/main/java/org/checkerframework/dataflow/analysis/AnalysisResult.java

```diff
@@ -195,6 +201,45 @@ private static void mergeTreeLookup(
         return merged;
     }
 
+    /**
+     * Returns true if the given tree is dead code (will never be executed at run time).
+     *
+     * <p>This method is conservative. If it returns true, the tree is dead code. If it returns
+     * false, the tree may or may not be dead code.
+     *
+     * @param tree a tree
+     * @return true if the tree is dead code
+     */
+    public boolean isDeadCode(Tree tree) {
+        // Return false for statements.
+        if (!TreeUtils.isExpressionTree(tree)) {
+            return false;
```

Comment:
Why are statements never dead code?

Reply:
`isDeadCode` is retrieving information about expressions from dataflow.  I didn't see an easy way to determine whether a statement is dead code, so `isDeadCode` conservatively returns false for them.  The spec of `isDeadCode` always permits returning false.  (I can put some subset, or all, of this in a comment.)

Reply:
That makes sense.  A comment here about dataflow have no information about statements would be helpful.  
Does dataflow also not have information about the kinds of trees in the switch below?